<!-- -*- mode: XML -*- -->

<launch>
  <!-- declare arguments with default values -->
  <!-- the path of config file -->
  <arg name="config_path" default="$(find seyond)/config.yaml" />


  <node pkg="nodelet" type="nodelet" name="seyond_nodelet" args="manager" output="screen"/>
  <node pkg="nodelet" type="nodelet" name="SeyondDriver"
        args="load seyond/SeyondDriver seyond_nodelet" required="true">
    <param name="config_path" value="$(arg config_path)"/>
  </node>
  <!-- rviz -->
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find seyond)/rviz/rviz.rviz" />
</launch>
